spec/unit/service_spec.rb

```diff
@@ -461,6 +463,60 @@ module Backend
             expect(Service.save!(svc_attrs).active?).to be_falsy
           end
         end
+
+        it 'returns true when the service does not have state in the DB' do
+          service_id = '9001'
+          Service.save!({state: :suspended, id: service_id, default_service: false})
+          Service.storage.del ThreeScale::Backend::Service.storage_key(service_id, 'state')
+          expect(Service.load_by_id(service_id).active?).to be true
+          Service.delete_by_id(service_id)
```

Comment:
clean up not required here,

in case it was needed, better to use ```after```

Reply:
I understand cleanup is required there because we are saving it into the database.

Regarding the after hook, I'll implement it.

Reply:
I've pushed new changes with the implementation of the after hook